step-8/README.adoc

```diff
@@ -106,32 +106,18 @@ include::src/main/java/io/vertx/guides/wiki/database/WikiDatabaseServiceImpl.jav
 
 === Working with a database connection
 
-When direct queries do not fit (e.g. when multiple queries must participate in the same transaction), you can acquire a database connection from the pool.
-All you have to do is calling `rxGetConnection` on the `JDBCClient`:
+When direct queries do not fit (e.g. when using session variables/temporary tables), you can generate a reactive flow from a single `SQLConnection`:
 
 [source,java,indent=0]
 ----
-Single<SQLConnection> connection = dbClient.rxGetConnection();
+include::src/main/java/io/vertx/guides/wiki/database/WikiDatabaseServiceImpl.java[tags=using-connection-helper]
 ----
+<1> borrows a connection from the pool and provides it in a callback.
+<2> returns a `Single` generated from execution of queries and transforming results.
 
-The method returns a `Single<Connection>` which you can easily transform with `flatMapXXX` to execute SQL queries:
+You may execute any number of queries and transform results with RxJava operators to generate the flow.
```

Comment:
Totally minor: I'd simplify the sentence by removing _to generate the flow_.

Reply:
Done